pkg/block/metadata/hash_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package metadata
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/thanos-io/thanos/pkg/testutil"
+)
+
+func TestHashSmoke(t *testing.T) {
```

Comment:
`smoke`

![](https://media1.giphy.com/media/wHB67Zkr63UP7RWJsj/giphy.gif)

Did not hear that word for some time... ;p Smoke test is valid naming though (: